src/backend/metal/src/window.rs

```diff
@@ -16,7 +16,7 @@ use core_foundation::base::TCFType;
 use core_foundation::string::CFString;
 use core_foundation::dictionary::CFDictionary;
 use core_foundation::number::CFNumber;
-use core_graphics::base::CGFloat;
+//use core_graphics::base::CGFloat;
```

Comment:
Are we keeping these in gfx for now? Does it break hal/quad on your Retina display?

Reply:
Excellent point! They are broken. How about we only apply the scale when initialized via `winit` and ignore it otherwise?